index.bs

```diff
@@ -97,16 +100,16 @@ To <dfn>Construct a DeviceOrientationSensor Object</dfn> the user agent must inv
 
 <div algorithm="to populate quaternion">
 The {{DeviceOrientationSensor/populateQuaternion()}} method must run these steps or their [=equivalent=]:
-    1.  If |array| parameter is not {{Float32Array}}, [=throw=] an
+    1.  If |buffer| parameter is not of type {{Float32Array}}, [=throw=] an
         "{{SyntaxError!!exception}}" {{DOMException}} and abort these steps.
-    1.  If |array| parameter is {{Float32Array}} with size other than four, [=throw=] an
+    1.  If |buffer| parameter is of type {{Float32Array}} with a size other than four, [=throw=] a
```

Comment:
"of type" is good in prose (and is proper English). It is clear the behaviour mirrors that of the "typeof" operator defined in ES.